recipes/fast-dds/all/conanfile.py

```diff
@@ -114,9 +114,8 @@ def _configure_cmake(self):
     def requirements(self):
         self.requires("tinyxml2/9.0.0")
         self.requires("asio/1.21.0")
-        self.requires("fast-cdr/1.0.22")
+        self.requires("fast-cdr/1.0.23")
         self.requires("foonathan-memory/0.7.1")
-        self.requires("boost/1.75.0")  # boost/1.76 is required by version 2.3.2, boost/1.75.0 required for 2.3.3 by Windows
```

Comment:
so it was a mistake?

Reply:
Well look at the fast-dds docs they say the dependency on boost was removed in 1.4.0 (1 major version ago) And it runs on my cross compiled target that doesn't have boost. So probably someone pushed some very old recipes to cci ?

Reply:
but there is still 2.3.2 & 2.3.3 in this recipe, correct?

Reply:
https://fast-dds.docs.eprosima.com/en/latest/notes/notes.html#version-1-4-0 would be what you need to reach the same conclusion

Reply:
> but there is still 2.3.2 & 2.3.3 in this recipe, correct?

Yes  2.3.2-2.3.3 and 2.3.4 are still here but they are all newer than 1.4.0 so it should be fine

Reply:
seems 2.5.0 is to modern for gcc 5 any way to deal with that ?

Reply:
> seems 2.5.0 is to modern for gcc 5 any way to deal with that ?

Updating "validate" to reflect what is needed